keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -467,6 +471,32 @@ to be added by the cluster manager upon creation. The value of the label
 MUST be the same as the name of the cluster manager. The purpose of this
 label is to make filter clusters from different cluster managers easier.
 
+#### Endpoints and access credentials
+
+Three fields provide the information needed to access the cluster's API endpoint.
+
+##### Endpoints
+
+A list of API endpoints. Having a list allows endpoints to roll over:
+when an endpoint needs to change, the new endpoint is added to the list,
+and when the old endpoint stops being available, it is removed. Clients
+need to try all listed endpoints until they find one which they can
+connect to.
+
+##### Provider
+
+The name of a credential provider. This is a value set by the cluster
+manager identifying how the credential information is supposed to be
+used.
+
+##### Credential object reference
+
+A reference to an external object providing access credentials. This
```

Comment:
what permission the credential will have? will it decided solely by the provider, or user need to request as a field in the spec?